app/src/main/java/com/khinthirisoe/bakingapp/ui/steps/StepsContract.kt

```diff
@@ -0,0 +1,21 @@
+package com.khinthirisoe.bakingapp.ui.steps
+
+import com.khinthirisoe.bakingapp.ui.steps.player.MediaPlayer
+
+interface StepsContract {
+
+    interface Presenter {
```

Comment:
@KhinThiriSoe shouldn't all your presenter be extending the BasePresenter

Reply:
should be extended BasePresenter?